src/realty/domain/realty-images.ts

```diff
@@ -0,0 +1,35 @@
+import { RealtyImagesErrorCodes } from './realty-images-error-codes';
+import { RealtyImagesException } from './realty-images.exception';
+
+const MAX_IMAGES = 5;
+
+interface RealtyImagesProps {
+  url: string;
+  isCover: boolean;
+}
+
+export class RealtyImages {
+  private images: RealtyImagesProps[] = [];
+
+  public setImage(image: RealtyImagesProps): void {
```

Comment:
A palavra `set` remete a  dizer algo do tipo `x = y`. Então, nesse caso, eu entenderia que A imagem dessa realty passaria a ser a que eu estou mandando. Porém não é o caso, a imagem que eu mando é adicionada a uma lista de imagens. Então sugiro alterar o nome do método para `add`
```suggestion
  public addImage(image: RealtyImagesProps): void {
```

Reply:
Resolvido em https://github.com/devphaelmoraes/realty/commit/9dcd283c921cd33423fd5dd7e6500ea37cb5f6bf